src/config.ts

```diff
@@ -247,12 +243,7 @@ export function createExtendsStampsConfig(
     )
   }
 
-  if (
-    assertBoolean(POSTAGE_EXTENDS_CAPACITY) &&
-    POSTAGE_EXTENDS_CAPACITY === 'true' &&
-    POSTAGE_USAGE_THRESHOLD &&
-    !assertDecimal(POSTAGE_USAGE_THRESHOLD)
-  ) {
+  if (POSTAGE_EXTENDS_CAPACITY === 'true' && POSTAGE_USAGE_THRESHOLD && !assertDecimal(POSTAGE_USAGE_THRESHOLD)) {
```

Comment:
You will never get `false` out of `assertDecimal()` because it is "assert" which throws an error upon wrong value, see the implementation ;-) So I would move this call from the `if` into the `if-branch` (eq. one line bellow).

Reply:
I had my moment of "what was I thinking with this one"